PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchType.swift

```diff
@@ -0,0 +1,140 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Sync
+import Combine
+import SharedPocketKit
+
+class SavesOnlineSearch {
```

Comment:
will separate these classes into their own files before putting PR in ready to review